.rodare.json

```diff
@@ -0,0 +1,42 @@
+{
+    "creators": [
+	{
+            "name": "Eckert, Carlchristian",
+            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf, TU Dresden"
```

Comment:
```suggestion
            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf, TU Dresden",
            "orcid": "0000-0002-6459-0842"
```

Reply:
If you want, it would be cool if you added my orcid-ID as well :)

Reply:
@slizzered Thx for the fedback, sure I will add your ORCID